Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/common/use-hash-url-params.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { createContext, useCallback, useContext, useMemo } from 'react';
+import { useLocation } from 'react-router-dom';
+
+import { IHaveChildrenProps } from '../../components/common/Props';
+
+interface IHashUrlParamsContext {
+    state: { params: unknown };
```

Comment:
Can the unknown type be avoided here? I am not sure but you can possibly use a different type to match the params property.